terraform/deployments/cluster-services/aws_auth_configmap.tf

```
@@ -61,3 +75,47 @@ resource "kubernetes_cluster_role_binding" "cluster_admins" {
     api_group = "rbac.authorization.k8s.io"
   }
 }
+
+resource "kubernetes_cluster_role_binding" "cluster_readonly" {
+  metadata {
+    name = "cluster-readonly"
+  }
+  role_ref {
+    api_group = "rbac.authorization.k8s.io"
+    kind      = "ClusterRole"
+    name      = "view"
+  }
+  subject {
+    kind      = "Group"
+    name      = "readonly"
+    api_group = "rbac.authorization.k8s.io"
+  }
+}
+
+resource "kubernetes_cluster_role" "read_crs_and_crbs" {
+  metadata {
+    name = "read_crs_and_crbs"
```

You have dashes in the object name for the rolebinding but underscores here - best keep them consistent.
```suggestion
    name = "read-crs-and-crbs"
```

---

ok will do